Packs/Core/Playbooks/playbook-External_Login_Password_Spray.yml

```diff
@@ -0,0 +1,1305 @@
+      description: Checks if the operating system reported by the agent of the machine
+        to which the users logged in is Windows or another OS.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "33"
+      Windows:
+      - "65"
+    separatecontext: false
+    conditions:
+    - label: Windows
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: alert
+                accessor: hostos
+                transformers:
+                - operator: uniq
+                - operator: FirstArrayElement
+            iscontext: true
+          right:
+            value:
+              simple: AGENT_OS_WINDOWS
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 680,
+          "y": 1910
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "65":
+    id: "65"
+    taskid: b6911013-8c0a-458c-8c31-77e98216dd96
+    type: condition
+    task:
+      id: b6911013-8c0a-458c-8c31-77e98216dd96
+      version: -1
+      name: Which logon type was used?
+      description: Checks the logon type (E.g., Network, RemoteInteractive, etc) used
+        in the attack.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "33"
+      RemoteInteractive:
+      - "73"
+    separatecontext: false
+    conditions:
+    - label: RemoteInteractive
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              complex:
+                root: Core.OriginalAlert._all_events.login_data
+                filters:
+                - - operator: isEqualString
+                    left:
+                      value:
+                        simple: Core.OriginalAlert._all_events.login_data.outcome
+                      iscontext: true
+                    right:
+                      value:
+                        simple: SUCCESS
+                    ignorecase: true
+                accessor: type
+                transformers:
+                - operator: uniq
+            iscontext: true
+          right:
+            value:
+              simple: RemoteInteractive
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 680,
+          "y": 2150
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "67":
+    id: "67"
+    taskid: 868e45e4-7156-42ff-8e69-98cdd60d2a07
+    type: regular
+    task:
+      id: 868e45e4-7156-42ff-8e69-98cdd60d2a07
+      version: -1
+      name: Terminate RDP Session
```

Comment:
Same question about multiple users, did we checked that on multiple users?

Reply:
Yes, checked without users, with one user, with 2 users where one should be logged off, etc.